iam.tf

```diff
@@ -26,7 +26,10 @@ resource "aws_iam_role_policy" "write_to_cw" {
     },
     {
         "Effect": "Allow",
-        "Action": "ec2:DescribeTags",
+        "Action": [
+          "ec2:DescribeTags",
+          "ec2:DescribeInstances"
```

Comment:
are you able to scope this to the instance requesting it?

Reply:
Not with ease at infra. I'll remove and move it to the instance level instead.